### ISSUE TITLE
qbec env add / qbec env del

### ISSUE THREAD

**Issue by kvaps** [splunk/qbec#47]
Need automation for adding and removing environments, with describing them in `environments` directory and `params.libsonnet`, `qbec.yaml` files

**Comment by gotwarlost** [comment]
since `qbec` doesn't have an opinion on how to create runtime parameters and where environment specific configuration is stored, it cannot reasonably update these. The best it can do is add an environment entry in `qbec.yaml`. The rest of the structure can be vastly different for every project.

**Comment by kvaps** [comment]
But it is doing that for the initial default environment. Isn't it?

**Comment by gotwarlost** [comment]
Yes and I want to change that. The generated code is too much machinery to get started. I want `qbec init` to produce something like the config documented at https://qbec.io/userguide/usage/runtime-params/#using-qbec-io-env

**Comment by kvaps** [comment]
Please keep it simple for end user.

Allow users to cusomize main structure is cool idea, but current example is really fine for the beginners who have no much experience with jsonnet.

**Comment by kvaps** [comment]
Maybe leave this hierarchy structure only for `init --with-example`?
I think it is good compromise to solve that, and keep simplicity for beginners :)
